test/models/config_test.rb

```diff
@@ -82,16 +82,16 @@ class ConfigTest < ActiveSupport::TestCase
       end
     end
 
-    describe '#start_day' do
-      it 'should return the day of week as a symbol' do
-        assert(Config.start_day.equal? :monday)
+    describe '#budget_bar_max' do
+      it 'should return an integer of 1_000 if unconfigured' do
+        assert(Config.budget_bar_max == 1_000)
```

Comment:
some small minitest-isms

```suggestion
        assert_equal 1_000, Config.budget_bar_max
```